code/src/Styles/ProgressBar.css

```diff
@@ -0,0 +1,38 @@
+.progress-bar-wrapper {
```

Comment:
I don't have a lot of remarks regarding your CSS, i think it's well stuctured and well indented.